### DISCUSSION TITLE
Программа не видит второй джйстик с одинваковым названием

### DISCUSSION THREAD

**Discussion by msslayer** [tjden88/JoyMapper#13] [General]
Добрый день!
В семействе джойстиков Virpil есть вариант подключения панели управления в порт базы джойстика.
В моем случае в порт базы VPC MongoosT50 CM3 включена панель VPC SharkA50.
На самом джойстике есть 5 осей, на панели еще 4 оси.
При подключении панели через джойстик винда видит это как одно устройство, но Windows 11 не видит в одном джойстике более 5 осей. В итоге винда создает в системе два джойстика с одинаковым названием и "лишние" оси присваиват второму джойстику.
В моем случае винда создала джойстик с номером 2 для работы "лишних 4-х осей", и джойстик №4 с таким же именем на котором присутсвуют все кнопки и основные 5 осей (x/y/z/rotx/roty). 
Программа при этом не видит джойстик №4 (возможно из-за одинакового имени с джойстиком №2)
Переименовать или поменять местами джойстики в винде похоже что невозможно.
В итоге не получается использовыать программу чтобы повесить бинды на нажатие джойстика №4.

**Comment by tjden88** [comment]
Добрый день!
Попробуйте обновить программу до этой версии:
[jm1_5_beta.zip](https://github.com/user-attachments/files/17089801/jm1_5_beta.zip)
Пожалуйста, напишите, решена ли проблема

**Comment by msslayer** [comment]
Добрый вечер!
В версии 1.5 Mapper видит второй джойстик, все отлично работает!
Не хватает функции сворачивать в трэй и если совсем обнаглеть, то
автозапуска при старте винды.
И как можно Вас поддержать, хочу перевести небольшую благодарность за Ваш
труд.

вс, 22 сент. 2024 г. в 18:11, tjden88 ***@***.***>:

**Reply by tjden88** [reply]
Функции автозапуска и сворачивания планируются, будут добавлены в следующем обновлении.
Кнопка доната находится в программе, раздел Настройки. Благодарю)
